src/Hackathon.Fiap.Web/Endpoints/Doctors/AppointmentConfiguration.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Ardalis.Result.AspNetCore;
+using Hackathon.Fiap.UseCases.Doctors.AppointmentConfiguration;
+
+namespace Hackathon.Fiap.Web.Endpoints.Doctors;
+
+/// <summary>
+/// Register a new doctor appointment configuration
+/// </summary>
+public partial class AppointmentConfiguration(IMediator mediator) : Endpoint<RegisterAppointmentConfigurationRequest>
+{
+    private readonly IMediator _mediator = mediator;
+
+    public override void Configure()
+    {
+        Put(RegisterAppointmentConfigurationRequest.Route);
+        AllowAnonymous();
```

Comment:
I think this should be limited to only doctors

Reply:
We can do that later, but we also need to validate that the doctor can only insert this to themselves